app/models/electronic_locations/online_location.rb

```diff
@@ -0,0 +1,11 @@
+module ElectronicLocations
+  # Model for electronic location subfield values for online locations
+  class OnlineLocation
```

Comment:
`ElectronicLocations::OnlineLocation` doesn't seem to be used anywhere except as the super-class of `ElectronicLocations::IIIFManifestURI` and `ElectronicLocations::Holding`.  It's only method is over-ridden by both of it's sub-classes. Is it needed for something? Future expansion?